rust/integration-tests/src/diff_tests/mod.rs

```diff
@@ -25,25 +25,37 @@ pub fn remove_transaction_timestamp(value: &mut Value) {
 }
 
 #[allow(dead_code)]
```

Comment:
Could we put all these in a `mod tests`

Reply:
this prevents us from importing : https://github.com/aptos-labs/aptos-indexer-processor-sdk/pull/73